c++/src/capnp/layout.c++

```diff
@@ -578,7 +578,8 @@ struct WireHelpers {
       const word* ptr = ref->farTarget(segment);
       auto padWords = (ONE + bounded(ref->isDoubleFar())) * POINTER_SIZE_IN_WORDS;
       KJ_REQUIRE(boundsCheck(segment, ptr, padWords),
-                 "Message contains out-of-bounds far pointer.") {
+                 "Message contains out-of-bounds far pointer."
```

Comment:
Minor, but it looks like none of these messages include a space between the terminating period of the specific message and `OUT_OF_BOUNDS_ERROR_DETAIL`. E.g. this message will contain `far pointer.This usually`.